docs/versions/README.md

```diff
@@ -0,0 +1,70 @@
+# Semantic Versioning for Releases
+
+## Motivation
+
+Semantic versions, ie, versions numbers in the form `X.Y.Z`, are useful for
+quickly comparing two sets of data and code and being able to tell their
+relative recency. This document outlines how semantic versions will be
+assigned to releases.
+
+## Background
+
+Currently, all releases are assigned a `kf_id` identifier with an `RE`
+prefix. Although this is enough to distinguish individual versions,
+it is not enough to tell whether where two versions sit in time
+or compatability.
+
+
+## Implementation
+
+
+### Release Versioning Rules
+
+The rules for versioning a release should be as follows:
+
+The maximum minor and patch versions will be used as the basis for the
+new version. That is, of all the studies in a release, the new minor version
+for the release will be the highest minor version among the release plus one
+and vice-versa for the patch version.
+There should never be disagreements in the major version number, unless one
+or more release have never been assigned a version.
+From this this base version, a new version is issued on the following:
+
+- A major release is issued for any release that includes all public studies
+- A minor release is issued for a release that is accepted
+- A patch release is issued for any release
+
+
+### Portal Versioning Rules
+
+An all-encompassing portal version is issued for every release based on
+the same criteria as above.
+
+## Examples
+
+### Patch versioning for review process
+
+For every candidate release between releases to the portal, the patch version will be bumped globally in the coordinator.
+Note that none of these relases except the first and last rows are published to the portal.
+
+| Notes                                      |  SD_1 |  SD_2 |  SD_3 | Data Version |
+|--------------------------------------------|:-----:|:-----:|:-----:|:------------:|
+| Initial Release                            | 1.0.0 | 1.0.0 | 1.0.0 |     1.0.0    |
+| First candidate release                    | 1.0.1 |   -   |   -   |     1.0.1    |
+| Second candidate release                   | 1.0.2 | 1.0.2 | 1.0.2 |     1.0.2    |
+| Third candidate release                    | 1.0.3 |   -   | 1.0.3 |     1.0.3    |
+| Fourth candidate release                   | 1.0.4 |   -   | 1.0.4 |     1.0.4    |
+| Successfully release of Fourth candidate   | 1.1.0 | 1.1.0 | 1.1.0 |     1.1.0    |
+
```

Comment:
Just to note a few things we discussed in person - each one of these still gets a `RE` ID. As described here, the public releases to the main portal will always be 0 in the patch slot but we'll still keep the three digits for consistency/future-proofing.